### ISSUE TITLE
Readme.md update

### ISSUE THREAD

**Issue by balaji1233** [JatinAgrawal0/SAYC#1]
Hey ! In readme.md you have written to replace the api key in app.py but it should be replace in  YoutubeCommentScrapper.py right?And  in what places do I have to add the api key in YoutubeCommentScrapper.py and what other changes I have to make in file beside adding api key

**Comment by SHIVAM4510** [comment]
i am getting error of streamlit that no secret file found

**Comment by JatinAgrawal0** [comment]
Yes, you're absolutely right – the API key should be replaced in YoutubeCommentScrapper.py and not app.py and you only have to update it in YoutubeCommentScrapper.py
